src/Chunk.cpp

```diff
@@ -741,6 +767,29 @@ void cChunk::MoveEntityToNewChunk(cEntity * a_Entity)
 
 
 
+void cChunk::UpdateCanUnload()
```

Comment:
The motive is the need to update that counter. I'll see if I can think of a less error prone solution.